report/Titelblad.tex

```diff
@@ -33,7 +33,16 @@
   \includegraphics[width=0.9\textwidth]{images/aau_logo.pdf}\\[1.0 cm]
  \end{flushright}
  \begin{flushleft}
-  \textbf{Abstract:}\\INSERT HERE
+  \textbf{Abstract:}\\
+  In the danish education system, the Arduino platform is often used to introduce the students to computer science. 
+  The Arduino's hardware is relatively simple simple and easy to grasp, however the Arduino language can often be difficult for beginners.
+  The Arduino language is a subset of C++, and while it is simplified compared to C++, there are still parts of the language that are difficult to comprehend for novices.
```

Comment:
maybe change to 
While the Arduino language is a simplified subset of C++, there are still ....
just to shorten a bit.